ntbs-service/Pages/Notifications/Edit/Patient.cshtml.cs

```diff
@@ -25,10 +24,11 @@ public class PatientModel : NotificationEditModelBase
         [BindProperty]
         [ValidFormattedDateCanConvertToDatetime(ErrorMessage = ValidationMessages.InvalidDate)]
         public FormattedDate FormattedDob { get; set; }
+        public IPostcodeService PostcodeService { get; set; }
```

Comment:
Shouldn't this be a private field rather than a property, as it's only used within this class?